content/08-mongodb/14-mongodb-encryption.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: 'How MongoDB encrypts data'
+metaTitle: "Understanding data encryption in MongoDB"
+metaDescription: "Learn how MongoDB encrypts data to ensure security in all database instances"
+metaImage: '/social/generic-mongodb.png'
+authors: ['alexemerich']
+---
+
+
+## Introduction
+Today’s world runs on data. The data that an organization creates, stores, and exchanges are its most valuable asset. Data encryption is a critical practice for all organizations for their protection, users’ protection, and compliance with data privacy regulations. An organization should make sure that internal users should be only able to view data relevant to them. Even more so, an organization needs to secure its data against compromise and unauthorized access. 
+
+The need for data encryption is even more paramount for organizations handling sensitive information. In this article, we discuss why you should encrypt your data in addition to robust [authentication](/intro/database-glossary#authentication) and [authorization](/intro/database-glossary#authorization) measures, the different types of encryption in [MongoDB](/intro/database-glossary#mongodb), and encryption best practices.
+
+<PrismaOutlinks>
+
+Prisma's [MongoDB connector](https://www.prisma.io/docs/concepts/database-connectors/mongodb) has recently been [promoted to general availability](https://www.prisma.io/blog/mongodb-general-availability-pixnun6mffmu)!  With this change, you can use the [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) to manage production MongoDB databases with confidence.
```

Comment:
Just a note that we probably want to switch this one out with something else before publication.  I don't think we've gotten much feedback on where to link to specifically yet, so your call.

Reply:
Yeah agreed. I may hold off until we get a bit more definitive answer or just remove the mention of launch week. Can delay until I have the image assets as well.